server/schema/addBook.js

```diff
@@ -0,0 +1,188 @@
+import Joi from '@hapi/joi';
+
+/**
+ * @description Book schema object for Joi validation
+ * @constant
+ */
+export default {
+  title: Joi.string()
+    .min(2)
+    .max(100)
```

Comment:
@tolulope-od , you did a great job for validating the client inputs for the book addition endpoint. For clarity purpose, what informed your decision to set 100 as the maximum character length for the book title?

Reply:
I just figured that book titles usually do not have very long names, but if this is something that needs reviewing it can be changed in another PR